### DISCUSSION TITLE
Get webcam stream

### DISCUSSION THREAD

**Discussion by fengbozs** [MicrosoftEdge/WebView2Feedback#3879] [Q&A]
Consider adding a video stream capture callback? This is very important in some special applications, such as adding face monitoring function in the application

**Comment by champnic** [comment]
Hey @fengbozs - could you describe a little more detail about what you'd like to see here? Do you mean if you setup webcam video on a webpage you'd like to have the host app do some processing?

I imagine this is a task better suited for the native host app to implement, or try to do it completely in HTML/JS using `MediaDevices.getUserMedia` and the `MediaStream`.
